data/governments.txt

```diff
@@ -896,6 +896,21 @@ government "Lunarium"
 		"Heliarch" -.01
 		"Pirate" -.01
 
+government "Lunarium (Hidden)"
+	swizzle 5
```

Comment:
Should this have a display name?

Reply:
It did, but then I messed it up when I committed to the wrong branch...